src/Dumplie/Application/Metadata/Schema/Type.php

```diff
@@ -33,6 +35,22 @@ public function __construct(string $type)
     }
 
     /**
+     * @return Type
+     */
+    public static function association(): Type
+    {
+        return new static(static::TYPE_ASSOCIATION);
```

Comment:
Actually its TYPE_ASSOCIATION_SINGLE, there is also a possibility that we are going to introduce association between multiple objects